File: shoebill_plugins/LtrpApi/src/main/java/lt/ltrp/event/job/PlayerChangeJobEvent.java
package lt.ltrp.event.job;



import lt.ltrp.event.player.PlayerEvent;
import lt.ltrp.object.Job;
import lt.ltrp.object.LtrpPlayer;

/**
 * @author Bebras
 *         2016.04.07.
 */
public class PlayerChangeJobEvent extends PlayerEvent {

    private Job oldJob;
    private Job newJob;

    public PlayerChangeJobEvent(LtrpPlayer player, Job oldJob, Job newJob) {
        super(player);
        this.oldJob = oldJob;
        this.newJob = newJob;
    }

    public Job getOldJob() {
        return oldJob;
    }

    public Job getNewJob() {
        return newJob;
    }
}

File: shoebill_plugins/player-job/src/main/java/lt/ltrp/player/job/event/PlayerChangeJobEvent.kt
package lt.ltrp.player.job.event

import lt.ltrp.`object`.LtrpPlayer
import lt.ltrp.event.player.PlayerEvent
import lt.ltrp.job.`object`.Job


/**
 * @author Bebras
 *         2016.04.07.
 */
class PlayerChangeJobEvent(player: LtrpPlayer, val oldJob: Job?, val newJob: Job?): PlayerEvent(player) {

}
